templates/cdhpages/blocks/standard_tile_block.html

```diff
@@ -0,0 +1,25 @@
+<h2>Standard tile block</h2>
+{% load wagtailcore_tags %}
+
+{% if self.heading %}
+{% include 'springkit/blocks/heading/jumplinkable_heading.html' %}
+{% endif %}
+
+<div>
+{% if self.description %}
+  {{ self.description }}
+{% endif %}
+</div>
+<div>
+{% if self.see_more_link.page %}
+<a href="{{ BASE_URL }}{{ self.see_more_link.url }}" class="standard-tiles__link">
```

Comment:
We shouldn't need `BASE_URL` here (and it's best to avoid in general, whenever we can, in favour of  something like `thing.get_absolute_url()`)